dev/registry.sh

```diff
@@ -25,6 +25,11 @@ source $ROOT/dev/util.sh
 
 GCR_HOST=${GCR_HOST:-"gcr.io"}
 
+# set variables to empty strings if they do not exist
```

Comment:
I don't think we really need this comment since this script is for us and it's obvious what it does. What do you think?